examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb

```diff
@@ -97,7 +97,11 @@
     "from dask_saturn import SaturnCluster\n",
```

Comment:
<p>can you add a short sentence about what <code>.scheduler_info</code>  and <code>.client</code>  will print, and why it matters? In my last round of cleanup on these notebooks, I tried to eliminate places where there were lines of code with 0 explanation. I think we should try for these to flow like a tutorial, where each step has an explanation. It doesn't have to be long, but just like the one sentence you would say over this line in a meetup talk or something.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I think these were leftover from some other copy/pasting, but I would prefer to remove them. I don't think it provides much for this particular example.</p>

Reply:
ok sure, even better

Reply:
ok I just updated to the latest version of all linters

```shell
pip install --upgrade black mypy flake8 pylint
```

then ran `make format` on your branch.

Nothing was changed! So maybe you had an older version of `black` or something.